CHANGELOG.md

```diff
@@ -1,3 +1,42 @@
+# 94.1.0
+
+This minor release intoduces new APIs to FlexibleHeader and Ripple, and updates MDCSnackbar appearance from legacy to 2018. The release also includes bug fixes for multiple components.
```

Comment:
Tip: A bit of wordsmithing here. I referred to "2018" mostly for PR feedback and am not sure it's the best description for the CHANGELOG. What do you think?